NuRadioMC/examples/01_Veff_simulation/T01generate_event_list.py

```diff
@@ -9,8 +9,8 @@
 'fiducial_rmin': 0 * units.km,
 'fiducial_rmax': 4 * units.km}
 
-# generate one event list at 1e19 eV with 1000 neutrinos
-generate_eventlist_cylinder('1e19_n1e3.hdf5', 1e3, 1e19 * units.eV, 1e19 * units.eV, volume)
+# generate one event list at 1e19 eV with 10000 neutrinos
+generate_eventlist_cylinder('1e19_n1e4.hdf5', 1e4, 1e19 * units.eV, 1e19 * units.eV, volume)
 
 # generate one event list at 1e18 eV with 10000 neutrinos
 generate_eventlist_cylinder('1e18_n1e4.hdf5', 1e4, 1e18 * units.eV, 1e18 * units.eV, volume)
```

Comment:
These events are not used, right? Shall we delete the generation?

Reply:
No strong feelings either way here - this doesn't take long to run, and it does show how to adjust the energy / allow someone to run T02 with different input files.